ipaplatform/suse/services.py

```diff
@@ -163,11 +162,33 @@ def is_running(self, instance_name="", wait=True):
         return False
 
 
+# For services which have no SUSE counterpart
+class SuseNoService(base_services.PlatformService):
+    @staticmethod
+    def start():
+        return True
+
+    @staticmethod
+    def stop():
+        return True
+
+    @staticmethod
+    def restart():
+        return True
+
+    @staticmethod
+    def disable():
+        return True
```

Comment:
We prefer regular methods

```suggestion
    def start(self):
        return True

    def stop(self):
        return True

    def restart(self):
        return True

    def disable(self):
        return True
```

Reply:
This was a direct copy from the Debian section.  I have applied your suggestion in the latest commit.